src/components/signin/Signin.tsx

```diff
@@ -0,0 +1,122 @@
+        selfIntro: 'I rather like baseball than curry.',
+        likedNum: 0,
+        isSecretMode: false,
+      },
+    }
+    // これ以降にresponseに対する処理を記載
+    // recoilにデータを保存して、画面遷移する
+    if (response.status === 200) {
+      const userId = Object.keys(response)[1]
+      const loginUserdata = {}
+      loginUserdata[userId] = response[userId]
+      console.log(loginUserdata)
+      setUserInfo(loginUserdata)
```

Comment:
ちょっと冗長な気がするので、下記でも良い気もします！
```suggestion
      setUserInfo(response[userId])
```

Reply:
@kan-mayoshi 短くできるということですね！こちらも直接確認させてください！